### DISCUSSION TITLE
Got a warning for using Sycnex Windows10Debloater, what do?

### DISCUSSION THREAD

**Discussion by JIVS** [Sophia-Community/SophiApp#67] [Q&A] [answered]
Basically that the stability of the OS was compromised by that script, something that TBH I neither knew or noticed

Should I use SophiApp anyway? or I'm about to destroy my current win10 install and lose everything?

**Comment by farag2** [comment, marked as answer]
Hi. No, you're not gonna lose anything. :) We just notice users that they used that script before that made harmful actions to Windows due to Sycnex is very far from coding, PowerShell and respecting users. All that he does with his so-called "debloating" script is destroying users' OS. look at his code and attitude to users: he even didn't archive his repo after discontinuing support. He just left it as is, placing a disclamer that he doesn't care at all what may happen with OS. And it has 12k stars!
So it's up to you now: continue working on the current OS, even applying SophiApp or re-install Windows and configure it from the beginning using safe and normal scripts or apps that configure Windows via official Microsoft ways.
As an example, [Sophia Script](https://github.com/farag2/Sophia-Script-for-Windows) and SophiApp do this.

**Reply by JIVS** [reply]
It didnt make my install any more unstable so I figured it was good enough, so I cant reverse the damage?

**Reply by farag2** [reply]
I do not now what you applied. If you think you didn't do anything important to your OS, it's OK. but a SophiApp created log will have a string that you accepted risk due to you applied or just launch Sycnex's script.

> I cant reverse the damage

It depends on what you chose in the script.

**Reply by JIVS** [reply]
I only used the default blocklist and then removed cortana and onedrive, no more no less

**Reply by farag2** [reply]
Hm, you're lucky to avoid using it then. :) Well, I do not see any barriers to use SophiApp. if you want to remove that warning, just remove folder `%SystemDrive%\Temp\Windows10Debloater` and never run such outdated, deprecated scripts without knowing what they do.

**Reply by farag2** [reply]
As you may see, the checking is very simple, but it works. Unfortunately users who apply such scripts do know know much about Windows (forgive me :cat:) and we can easily track whether this script was even just launched.

**Reply by farag2** [reply]
And feel free to bug report!
https://github.com/Sophia-Community/SophiApp/releases/tag/1.0.0.59

**Reply by farag2** [reply]
https://github.com/Sophia-Community/SophiApp#system-requirements

**Comment by farag2** [comment]
That's why Sophia family projects were born: https://github.com/Sophia-Community/SophiApp#about-sophiapp
